Fixing build with gcc 4.5. 
Upstream fix e22d2ed95925d870876c4fcd922ea2a9cfdbdab1

http://bugs.gentoo.org/show_bug.cgi?id=321411

--- ginac/function.pl
+++ ginac/function.pl
@@ -1314,7 +1314,7 @@
 	
 	// No derivative defined? Then return abstract derivative object
 	if (opt.power_f == NULL)
-		return (new power::power(*this, power_param))->setflag(status_flags::dynallocated |
+		return (new GiNaC::power(*this, power_param))->setflag(status_flags::dynallocated |
 	                                               status_flags::evaluated);
 
 	current_serial = serial;
